### DISCUSSION TITLE
Listening on different host or IP address

### DISCUSSION THREAD

**Discussion by YpNo** [directus/directus#11489] [Feature Requests]
Hi,

When I've tried directus, I was suprised that I couldn't chose on which interface directus should listen on. I don't want to directly expose my directus instance on the Internet.
I would like to configure that with a parameter config.

Regards.

**Comment by rijkvanzanten** [comment]
This was added in https://github.com/directus/directus/pull/11465 (thanks again @YpNo! ❤️)
